i18n/en.json

```diff
@@ -654,9 +654,9 @@
   "admin.group_settings.group_details.add_team_or_channel": "Add Team or Channel",
   "admin.group_settings.group_details.group_profile.name": "Name:",
   "admin.group_settings.group_details.group_teams_and_channels_row.remove": "Remove",
-  "admin.group_settings.group_details.group_teams_and_channels_row.remove.confirm_body": "Removing this membership will prevent future users in this group from being added to the '{name}' {displayType}.",
+  "admin.group_settings.group_details.group_teams_and_channels_row.remove.confirm_body": "Removing this membership will prevent future users in this group from being added to the ''{name}'' {displayType}.",
```

Comment:
where are the changes in the code for this one

Reply:
Updated